docs/kubernetes/windows_demo.md

```diff
@@ -0,0 +1,227 @@
+    ```
+
+4. On one of the Windows nodes, create an SMB share.
+    
+    Remember the name of this node, will be needed next step.
+    ```Powershell
+    $Password = ConvertTo-SecureString "<your password>" -AsPlainText -Force
+    New-LocalUser -Name smbuser -AccountNeverExpires -Password $Password
+    Add-LocalGroupMember -Group "Remote Desktop Users" -Member smbuser
+    New-Item -ItemType "directory" -Path "C:\SMBShare"
+    New-SmbShare -Name "SMBShare" -Path "C:\SMBShare" -FullAccess "smbuser"
+    ```
+
+5. Update necessary YAML files in Filestore driver workspace.
+    
+    Information you need:
+    - Project name
+    - Name of the node that hosts the SMB share
+    - Password from previous step
+    - Name of the nodes you will be deploying to.
+
+    List of files:
+    - `deploy/kubernetes/manifests/node_windows.yaml`
+    - `examples/kubernetes/windows/secrets.yaml`
+    - `examples/kubernetes/windows/pv.yaml`
+    - `examples/kubernetes/windows/nettest-pod.yaml`
+    - `examples/kubernetes/windows/powershell-nettest-pod.yaml`
+
+## Deploying
+[//]: # (Use kubelet commands instead of logging in)
+1. Log in to the nodes you specified in `examples/kubernetes/windows/nettest-pod.yaml` and `examples/kubernetes/windows/powershell-nettest-pod.yaml`.
+2. Run the filestore driver.
+
+    The driver binary has to be run manually on Windows as there is no support for privileged containers. As a result, [CSI plugins don't work on Windows](https://github.com/kubernetes/enhancements/blob/master/keps/sig-windows/20190714-windows-csi-support.md). 
+
+    ```Powershell
+    $PROJECT = '<your project>'
+    cd C:\
+    gsutil cp gs://$PROJECT-bucket/gcfs-csi-driver.exe .
+    mkdir C:\var\lib\kubelet\plugins\filestore.csi.storage.gke.io
+    .\gcfs-csi-driver.exe --v=5 --endpoint="unix:/var/lib/kubelet/plugins/filestore.csi.storage.gke.io/csi.sock" --nodeid="(hostname)" --node=true
+    ```
+3. Back from the Filestore driver workspace, run the *daemonset* for the node-driver-registrar container.
+    ```bash
+    ./deploy/kubernetes/driver_start_windows.sh
+    ```
+4. Finally apply all the YAML files.
+    ```bash
+    ./examples/kubernetes/windows/apply.sh
+    ```
+
+## Demo
+0. Commands to help track status of pods:
+    ```bash
+    kubectl get pods
+    kubectl describe pod nettest
+    kubectl describe pod powershell-nettest
+    ```
+1. Run each of the following commands in a different terminal.
+    ```bash
+    kubectl exec -it nettest powershell.exe
+    kubectl exec -it powershell-nettest -c powershell pwsh.exe
+    kubectl exec -it powershell-nettest -c nettest powershell.exe
+    ```
+
+2. From one of the containers, run:
+    ```Powershell
+    Set-Content -Path ".\smbshare\Hello-Containers.txt" -Value "Hello World!" 
+    ```
+
+3. And from one of the other containers:
+    ```Powershell
+    Get-Content -Path ".\smbshare\Hello-Containers.txt"
+    ```
+
+## Tearing down
+1. Delete the YAML files.
```

Comment:
This doesn't delete the YAML files, it un-applies them; it deletes the Kubernetes pods/resources. Please adjust.